chart/templates/csi-daemonset.yaml

```
@@ -43,7 +43,9 @@ spec:
             fieldRef:
               fieldPath: status.podIP
         - name: RUST_BACKTRACE
-          value: "1"
+          value: "1"{{ if .Values.moac }}
+        - name: MOAC
+          value: "true"{{ end }}
```

```suggestion
          value: "1"
{{ if .Values.moac }}
        - name: MOAC
          value: "true"
{{ end }}
```

`if` in helm can be written in the following format to make it more readable.

---

This is a merge request, I only went as far as making it build and pass pre-commit and CI tests. Feel free to raise another PR to fix this.